src/lighteval/tasks/prompt_manager.py

```diff
@@ -308,6 +308,13 @@ def get_examples(  # noqa: C901
         else:
             examples.append(content)
 
+        # Assistant prefix
```

Comment:
would have been needed for this PR for the original template, can remove as it's not needed now